azure/services/managedclusters/spec.go

```diff
@@ -255,6 +255,7 @@ func (s *ManagedClusterSpec) Parameters(existing interface{}) (params interface{
 			dnsIP := ip.String()
 			managedCluster.NetworkProfile.DNSServiceIP = &dnsIP
 		} else {
+			managedCluster.NetworkProfile.ServiceCidr = &s.ServiceCIDR
```

Comment:
We can keep this line outside of if else condition right ? 

To avoid redundancy ??